RecoLocalTracker/SubCollectionProducers/src/JetCoreClusterSplitter.cc

```diff
@@ -215,8 +220,8 @@ bool JetCoreClusterSplitter::split(
 // order with fast algo and pick first and second instead?
 std::pair<float, float> JetCoreClusterSplitter::closestClusters(
     const std::vector<float>& distanceMap) {
-  float minDist = 9e99;
-  float secondMinDist = 9e99;
+  float minDist = std::numeric_limits<float>::max();
+  float secondMinDist = std::numeric_limits<float>::max();
```

Comment:
took the opportunity to remove some numeric nonsense and clean math a bit.
This code requires a major cleanup....